src/Codeception/Lib/Driver/Db.php

```diff
@@ -221,7 +221,20 @@ public function executeQuery($query, array $params)
             throw new \Exception("Query '$query' can't be prepared.");
         }
 
-        $sth->execute($params);
+        $i = 0;
+        foreach ($params as $value) {
+            $i++;
+            if (is_bool($value)) {
+                $type = \PDO::PARAM_BOOL;
+            } elseif (preg_match('/^[-+]?\d+$/', $value)) {
+                $type = \PDO::PARAM_INT;
```

Comment:
I wonder if this could cause an issue if someone is trying to store a number in text column.
Probably is_int check would be more appropriate.